submissions/DmytroMukhin/HTML_CSS_Hooli-style_Popup/index.html

```diff
@@ -0,0 +1,238 @@
+                            height="30"
+                          />
+                          <figcaption class="popup__figcaption">Photo</figcaption>
+                        </figure>
+                      </a>
+                    </li>
+                    <li class="popup__item">
+                      <a class="popup__link" href="#">
+                        <figure class="popup__figure">
+                          <img
+                            class="popup__img"
+                            src="icons/apps.png"
+                            alt=""
+                            width="40"
+                            height="30"
+                          />
+                          <figcaption class="popup__figcaption">Apps</figcaption>
+                        </figure>
+                      </a>
+                    </li>
+                    <li class="popup__item">
+                      <a class="popup__link" href="#">
+                        <figure class="popup__figure">
+                          <img
+                            class="popup__img"
+                            src="icons/documents.png"
+                            alt=""
+                            width="40"
+                            height="30"
+                          />
+                          <figcaption class="popup__figcaption">
+                            Documents
+                          </figcaption>
+                        </figure>
+                      </a>
+                    </li>
+                    <li class="popup__more">
+                      <input
+                        class="popup__input--more"
+                        name="input-popup-more"
+                        id="popup__input--more"
+                        type="checkbox"
+                      />
+                      <label class="popup__label--more" for="popup__input--more"
+                        >more</label
+                      >
+                      <label class="popup__label--less" for="popup__input--more"
+                        >less</label
+                      >
+                      <ul class="popup__list--more">
+                        <li class="popup__item popup__item--hiden">
+                          <a class="popup__link" href="#">
+                            <figure class="popup__figure">
+                              <img
+                                class="popup__img"
+                                src="icons/chat.png"
+                                alt=""
+                                width="40"
+                                height="30"
+                              />
+                              <figcaption class="popup__figcaption">
+                                Chat
+                              </figcaption>
+                            </figure>
+                          </a>
+                        </li>
+                        <li class="popup__item popup__item--hiden">
```

Comment:
why do you need class `popup__item--hiden`?

Reply:
sory, I forgot to remove.